simulation/src/main/java/com/palantir/dialogue/core/Benchmark.java

```diff
@@ -86,6 +86,11 @@ static Benchmark builder() {
         return new Benchmark();
     }
 
+    public Benchmark mutableRequests() {
+        requestSupplier = Benchmark::constructNewRequest;
+        return this;
+    }
```

Comment:
Is this used?

Reply:
Not yet, thought I'd reduce the diff. Can remove if you want, but this is necessary for the other sticky implementation that pushes attachments around.

Reply:
I see, perhaps since we've already added request attachments, we should always use mutable request objects to prevent leaking data between requests?

Reply:
That would work, although I suspected this was done because the simulations were melting laptops. Lemme try with this by default.

Reply:
Alright, laptop didn't explode let's see CI.